applications/StructuralMechanicsApplication/python_scripts/structural_mechanics_solver.py

```diff
@@ -495,14 +496,16 @@ def _create_newton_raphson_strategy(self):
                                                                      self.settings["compute_reactions"].GetBool(),
                                                                      self.settings["reform_dofs_at_each_step"].GetBool(),
                                                                      self.settings["move_mesh_flag"].GetBool())
+        strategy.SetUseOldStiffnessInFirstIterationFlag(self.settings["use_old_stiffness_in_first_iteration"].GetBool())
+        return strategy
 
     def _create_line_search_strategy(self):
         computing_model_part = self.GetComputingModelPart()
         mechanical_scheme = self.get_solution_scheme()
         linear_solver = self.get_linear_solver()
         mechanical_convergence_criterion = self.get_convergence_criterion()
         builder_and_solver = self.get_builder_and_solver()
-        return KratosMultiphysics.LineSearchStrategy(computing_model_part,
+        strategy = KratosMultiphysics.LineSearchStrategy(computing_model_part,
```

Comment:
why this change?

Reply:
because i need to call the constructor and then the set function.

Reply:
Idem as my previous comment, and see @philbucher  comment

Reply:
If i change the constructor, i will break all current usages in which this is not passed.

As i understand the idea is to pass soon to the factory and hence to passing the parameters directly, so i wouldn't propagate this change through the code to then change it agan as soon as we pass to the parameters construction. 
This is particularly true since we are very short away from the Release, so i really would avoid pervasive changes.

Reply:
(note that using it as default is not enough, since all the previous parameters are given which forces to prescribe the last one)